bypros/test/docker-entrypoint-initdb.d/schema.sql

```diff
@@ -0,0 +1,395 @@
+--
```

Comment:
This is a dump and it should probably be a clean SQL schema instead, with higher expressivity and clearly indicating which fields are primary and foreign keys, and which are the indices, directly when defining the table.